views/js/runner/plugins/tools/highlighter/highlighter.js

```diff
@@ -87,6 +87,14 @@ define([
             }
         });
 
+        // same as previous but for touch devices (as from TAO-6578)
+        $(document).on('touchend.highlighter', function() {
+            if (isHighlighting && !selection.isCollapsed) {
+                highlightHelper.highlightRanges(getAllRanges());
+                selection.removeAllRanges();
+            }
```

Comment:
You duplicated this callback function. Please attach multiple events or move it to separate function.
e.g.:
```
$(document).on('touchend.highlighter mouseup.highlighter', function() {
```